### PR TITLE
Created Movie Tiles,  tomorrow will work with media queries

### PR DESCRIPTION
**Pull request by AdrianJar**

### CODE REVIEW

**Comment by ghost** [comment]
Great! I will be able to use the MovieList component in my code snippet

**Review by KatarzynaZaloba — APPROVED** [review]
Just an importing and renaming component, so ok ;)

File: src/feature/MovieBrowser/MoviesList/index.js
Comment (KatarzynaZaloba) [inline src/feature/MovieBrowser/MoviesList/index.js:1]: Great so far!

**Review by Marianna-Weychan — APPROVED** [review]
I added too white color :D But this is no problem, 
we'll merge later, it'll even out

**Review by KatarzynaZaloba — APPROVED** [review]

**Review by Marianna-Weychan — APPROVED** [review]
Great job Adrian!

**Review by Marianna-Weychan — APPROVED** [review]

**Review by Marianna-Weychan — APPROVED** [review]

**Review by KatarzynaZaloba — APPROVED** [review]

**Review by KatarzynaZaloba — APPROVED** [review]

**Pull request merged**